rotkehlchen/history/events/structures/evm_event.py

```diff
@@ -259,6 +260,13 @@ def __repr__(self) -> str:
         ]
         return f'EvmEvent({", ".join(fields)})'
 
+    def __str__(self) -> str:
+        return (
+            f'{self.event_type} / {self.event_subtype} EvmEvent in {self.location} with '
+            f'tx_hash={self.tx_hash.hex()} and time '
+            f'{timestamp_to_date(ts_ms_to_sec(self.timestamp))} using {self.asset}'
+        )
```

Comment:
will this override the `__repr__` in tests errors? It might make more difficult to compare events in tests

Reply:
Why would it? I am only touching the __str__ function here